### PR TITLE
Added delete button and functionality to UserDefaults tool

### PR DESCRIPTION
**Pull request by nikolamajcen**
## Summary

Provides the ability to delete a UserDefaults key. This makes it easy to test features dependent on certain flags, both for developers and especially for QA.

### Type

- [X] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

### Description

Added a delete button to the UserDefaults details screen, and made the screen pop from the navigation stack upon deletion. Also made the list sorted alphabetically so that keys can be found easily.
## Checklist

- [X] I have performed a self-review of my own code.
- [X] I have tested my changes, including edge cases.
- [X] I have added necessary tests for the changes introduced (if applicable).
- [X] I have updated the documentation to reflect my changes (if applicable).

## Additional notes

@dimovskidamjan, due to ongoing changes for release of version `2.0.0`, I've cherry-picked you changes for version that were submitted via #38. Please review this PR and if everything is okay, this will go out with `1.2.2` release.

### CODE REVIEW

File: Sentinel/Classes/UserDefaults/UserDefaultsTool.swift [hunk 1]
```diff
@@ -46,6 +46,8 @@ private extension UserDefaultsTool {
                 viewController.navigationController?.pushViewController(userDefaultsViewController, animated: true)
             }
         }
+        .sorted { $0.title < $1.title }
```
Comment (ZvonimirMedak) [inline Sentinel/Classes/UserDefaults/UserDefaultsTool.swift:49]: Maybe we could think about adding a search bar as well? Not for this release, but somewhere in the future

Comment (nikolamajcen) [reply Sentinel/Classes/UserDefaults/UserDefaultsTool.swift:49]: I do agree with that 👍 Feel free to add that to our board.

**Review by dimovskidamjan — APPROVED** [review]
